Gemfile.lock

```diff
@@ -245,33 +247,34 @@ GEM
     method_source (1.0.0)
     mime-types (3.3.1)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2019.1009)
-    mimemagic (0.3.4)
+    mime-types-data (3.2020.0512)
+    mimemagic (0.3.5)
```

Comment:
### [mimemagic](https://github.com/minad/mimemagic)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.3.4 → 0.3.5

[_[change-log](https://github.com/minad/mimemagic/blob/master/CHANGELOG.md), [source-code](https://github.com/minad/mimemagic)_]





<details>
<summary>Commits</summary>

A change of **2** commits. See the full changes on [the compare page](https://github.com/minad/mimemagic/compare/v0.3.4...v0.3.5).

These are the individual commits:
- (51bdc84) [Mimetype extensions are now ordered by freedesktop.org's pri…](https://github.com/minad/mimemagic/commit/51bdc8407811a95d301e6a63dd93b30e7430cb80)
- (01f92d8) [v0.3.5](https://github.com/minad/mimemagic/commit/01f92d86d15d85cfd0f20dabd025dcbd36a8a60f)


</details>